spec/support/active_record_classes.rb

```diff
@@ -1,339 +1,21 @@
-end
-
-FileUtils.rm('data.sqlite3') if File.exist?('data.sqlite3')
-
-ActiveRecord::Base.logger = Logger.new($stdout)
-ActiveRecord::Base.logger.level = Logger::WARN
-ActiveRecord::Base.establish_connection(
-  'adapter' => defined?(JRUBY_VERSION) ? 'jdbcsqlite3' : 'sqlite3',
-  'database' => 'data.sqlite3',
-  'pool' => 5,
-  'timeout' => 5000
-)
-
-ActiveRecord::Base.raise_in_transactional_callbacks = true if ActiveRecord::Base.respond_to?(:raise_in_transactional_callbacks)
-
-ActiveRecord::Schema.define do
-  create_table :fruits do |t|
-    t.string :name
-  end
-  create_table :tasks do |t|
-    t.string :title
-  end
-  create_table :vegetables do |t|
-    t.string :name
-  end
-  create_table :songs do |t|
-    t.string :name
-    t.string :artist
-    t.boolean :released
-    t.boolean :premium
-  end
-  create_table :cats do |t|
-    t.string :name
-  end
-  create_table :dogs do |t|
-    t.string :name
-  end
-  create_table :people do |t|
-    t.string :first_name
-    t.string :last_name
-    t.integer :card_number
-  end
-  create_table :movies do |t|
-    t.string :title
-  end
-  create_table :restaurants do |t|
-    t.string :name
-    t.string :kind
-    t.text :description
-  end
-  create_table :products do |t|
-    t.string :name
-    t.string :href
-    t.string :tags
-    t.string :type
-    t.text :description
-    t.datetime :release_date
-  end
-  create_table :colors do |t|
-    t.string :name
-    t.string :short_name
-    t.integer :hex
-  end
-  create_table :namespaced_models do |t|
-    t.string :name
-    t.integer :another_private_value
-  end
+ar_schema.instance_exec do
   create_table :uniq_users, id: false do |t|
     t.string :name
   end
   create_table :nullable_ids
-  create_table :nested_items do |t|
-    t.integer :parent_id
-    t.boolean :hidden
-  end
-  create_table :posts do |t|
-    t.string :title
-  end
-  create_table :comments do |t|
-    t.integer :post_id
-    t.string :body
-  end
   create_table :mongo_documents do |t|
     t.string :name
   end
-  create_table :books do |t|
-    t.string :name
-    t.string :author
-    t.boolean :premium
-    t.boolean :released
-    t.string :genre
-  end
   create_table :ebooks do |t|
     t.string :name
     t.string :author
     t.boolean :premium
     t.boolean :released
   end
-  create_table :disabled_booleans do |t|
-    t.string :name
-  end
-  create_table :disabled_procs do |t|
-    t.string :name
-  end
-  create_table :disabled_symbols do |t|
-    t.string :name
-  end
   create_table :encoded_strings
-  unless OLD_RAILS
-    create_table :enqueued_documents do |t|
-      t.string :name
-    end
-    create_table :disabled_enqueued_documents do |t|
-      t.string :name
-    end
-  end
-  create_table :conditionally_enqueued_documents do |t|
-    t.string :name
-    t.boolean :is_public
-  end
-  create_table :misconfigured_blocks do |t|
-    t.string :name
-  end
-  if defined?(ActiveModel::Serializer)
```

Comment:
Removed this check since we do not use Rails versions before 6 in our test suite.

Once again let me know if I should add it again.